.env.sample

```diff
@@ -6,3 +6,6 @@ DB_NAME_TEST=
 DB_PORT=
 DATABASE_URL=
 SECRET_KEY=
+SENDGRID_API_KEY =
```

Comment:
remove the spaces preceding the `=`

Reply:
Thank you. I have removed the free spaces in .env.sample.